src/views/app/TheIntegrations/index.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import Pagoplux from './components/PagoPlux.vue';
+</script>
+
+<template>
+  <div class="principal-container">
+    <h3 class="principal-container-title">
+      Integraciones disponibles
```

Comment:
This is not necessary, is enough with the title on the page

Reply:
oki